libs/coin-modules/coin-cosmos/src/bridge/js.ts

```diff
@@ -0,0 +1,128 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+import { getCryptoCurrencyById } from "@ledgerhq/cryptoassets";
```

Comment:
Another things that we can do right now is to sort import.

Reply:
Done here https://github.com/LedgerHQ/ledger-live/commit/2b1464b2964727932f601ad60cefd663626966c0